### DISCUSSION TITLE
Práctica de la pastelería

### DISCUSSION THREAD

**Discussion by KarlaILM** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#205] [Show and tell]
Hola compañeros, les comparto mi práctica, pueden verla en el siguiente enlace

https://karlailm.github.io/HTML-MisionFrontend-/

**Comment by JosePabloGrr** [comment]
@KarlaILM Felicidades por tu practica :) Todo muy bonito, como sugerencia al hacer "Hover" sobre un producto todo el contenido se desplaza por que incrementaste las medidas de tu .tarjeta-menu...
Para no mover todo el contenido en lugar de utilizar medidas fijas (heigh/width) lo ideal es que utilices:
.tarjeta-menu:hover {
    transform: scale(1.05);
}
.tarjeta-menu {
    transition: transform 0.1s;
}
Así la tarjeta crece, pero no mueve todo el contenido y estéticamente se ve mejor 💜 espero ayudar y gracias por compartir tu proyecto..
![MouseHover](https://user-images.githubusercontent.com/16726985/156469958-9a83fc87-70c2-4ff8-98b3-c6d796b7a143.gif)

**Reply by JuanSalvador73** [reply]
Muy bueno. Solo checa los estilos - veo que los manejas - en la parte del formulario cuando ves el sitio en móvil, ya que se traslapan unos a otros. Solo es cuestión de revisar la parte "responsiva". 

Por lo demás muy bien :D

**Comment by DanyVeneno** [comment]
Excelente karla  se ve bien saludos

**Comment by hackit92** [comment]
Está muy bonita tu página, tiene una paleta de colores muy agradable y cumple de sobra los requerimientos 10/10

**Comment by aleycolen98** [comment]
Muy bonita, sobre todo la paleta de colores es bastante agradable y combina a la perfección con el tema, felicidades!

**Comment by VictorManuelSalas** [comment]
Muy linda =)
